templates/static/product-detail.twig

```diff
@@ -33,7 +33,7 @@
                     Steven D. Paulson & Scott L. Keith
                 </div>
                 <div class="price | text-slate-light text-lg | mt-2" itemprop="offers" itemscope itemtype="http://schema.org/Offer">
-                    <meta itemprop="priceCurrency" content="USD" />
+                    <meta content="USD" itemprop="priceCurrency"/>
```

Comment:
@lindseybradford - just curious, what is your preferred way of ordering attributes?

Reply:
alpha?

Reply:
Good question. Generally, I try to order js-hooks first (if any), then id, class, name, value, styles. Unusual suspects like this though…alpha makes sense to me for that scenario!